scripts.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+  console.log('hey');
+
+  var hangmanWords = [
+ "the","of","and","a","to","in","is","you","that","it","he",
+ "was","for","on","are","as","with","his","they","i","at","be",
+ "this","have","from","or","one","had","by","word","but","not",
+ "what","all","were","we","when","your","can","said","there",
+ "use","an","each","which","she","do","how","their","if","will",
+ "up","other","about","out","many","then","them","these","so",
+ "some","her","would","make","like","him","into","time","has",
+ "look","two","more","write","go","see","number","no","way",
+ "could","people","my","than","first","water","been","call",
+ "who","oil","its","now","find","long","down","day","did","get",
+ "come","made","may","part"
+];
```

Comment:
It makes sense, it could be loaded via it's own script file, via a JSON object, or refactored to make a AJAX call to a DB and return a word.   Many options available.